ingestion/src/metadata/ingestion/source/pipeline/airflow.py

```diff
@@ -97,6 +105,11 @@ def __init__(
 
         # Create the connection to the database
         self._session = None
+        self.status_map = {
```

Comment:
this might be better suited to be a constant in the same module. STATUS_MAP = ...